Docs for installing python API in M1 (#9176)
cvc5/cvc5: docs/api/python/python.rst

You can compare examples using the two APIs by visiting the :doc:`examples page <../../examples/quickstart>`.

> Installation (Except MacOS with Apple Silicon)

Reviewer: ```suggestion
Installation (x86-64 variants of Linux and macOS)
```
The instructions don't work for Windows or ARM versions of Linux, so it is better to explicitly list what's supported.

------------

The base and pythonic Python API can be installed via `pip` as follows:

.. code:: bash

  pip install cvc5

> Installation (MacOS with Apple Silicon)
> ------------

Reviewer: ```suggestion
Installation (ARM64 variants of Linux and macOS)
------------
```

> For M1 Macs, the base and pythonic Python API can be installed from source as follows:

Reviewer: ```suggestion
For ARM64-based machines (including Apple computers with M1 and M2 chips), the base and the pythonic Python API can be installed from source as follows:
```
